src/components/navigation/NavBar.jsx

```diff
@@ -75,48 +48,71 @@ function NavBar({ menuBtn = false }) {
         </div>
         <div className="nav-bottom ">
           <ul className="nav-bottom__list">
-            <li className="nav-bottom__list-item change-nav-display">
-              <Link to="/products-page" className="btn nav-bottom-btn text">
-                Turbo-chargers
-              </Link>
-            </li>
-            <li className="nav-bottom__list-item change-nav-display">
-              <Link
-                to="/products-page"
-                className="btn btn-sec nav-bottom-btn text"
-              >
-                Oils Lubricants
-              </Link>
-            </li>
-            <li className="nav-bottom__list-item change-nav-display">
-              <Link
-                to="/products-page"
-                className="btn btn-sec nav-bottom-btn text"
-              >
-                Three-wheelers
-              </Link>
-            </li>
-            <li className="nav-bottom__list-item change-nav-display">
-              <Link
-                to="/products-page"
-                className="btn btn-sec nav-bottom-btn text"
-              >
-                Spare-Parts
-              </Link>
-            </li>
-            <li className="nav-bottom__list-item change-nav-display">
-              <Link
-                to="/products-page"
-                className="btn btn-sec nav-bottom-btn text"
-              >
-                Power Stations
-              </Link>
-            </li>
+            {pathname === "/products-page" ? (
+              <li className="nav-bottom__list-item">
+                <div className="nav-bottom-search-box">
+                  <input
+                    type="search"
+                    className="nav-bottom-search"
+                    placeholder=" search"
+                    onChange={(e) => {
+                      filterManager({
+                        type: "SEARCH",
+                        payload: e.target.value,
+                      });
+                    }}
+                  />
+                </div>
+              </li>
+            ) : (
+              <>
+                <li className="nav-bottom__list-item change-nav-display">
+                  <Link to="/products-page" className="btn nav-bottom-btn text">
+                    Turbo-chargers
+                  </Link>
+                </li>
+                <li className="nav-bottom__list-item change-nav-display">
```

Comment:
suggestion: you can create separate data files and loop around them to avoid code repetition.